packages/compile-solidity-tests/test/supplier.ts

```diff
@@ -224,9 +227,10 @@ describe("CompilerSupplier", function () {
           "Version8Pragma",
           compilations[0].contracts
         ); //update when necessary
-        assert(VersionLatestPragma.compiler.version.includes("0.8.")); //update when necessary
-        assert(
-          VersionLatestPragma.contractName === "Version8Pragma", //update when necessary
+        assert.isTrue(VersionLatestPragma.compiler.version.includes("0.8.")); //update when necessary
```

Comment:
I would also suggest changing this to use `assert.include`.  I don't think it makes sense to use `assert.isTrue` here; that's more for, like, checking that a particular boolean has been computed correctly.  For a weird custom assertion, it makes more sense to just use `assert`.  But in this case there's no need to do that, because there's `assert.include`. :)